<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toolbar_selector</class>
 <widget class="QWidget" name="toolbar_selector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>100</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin"><number>6</number></property>
   <property name="topMargin"><number>6</number></property>
   <property name="rightMargin"><number>6</number></property>
   <property name="bottomMargin"><number>6</number></property>
   <item row="0" column="0">
    <widget class="QToolButton" name="toolButton_polygonalSelector">
     <property name="text"><string>Polygonal selector</string></property>
     <property name="checkable"><bool>true</bool></property>
     <property name="toolButtonStyle"><enum>Qt::ToolButtonTextBesideIcon</enum></property>
     <property name="autoRaise"><bool>true</bool></property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QRadioButton" name="radioButton_showInside">
     <property name="text"><string>Show inside</string></property>
     <property name="checked"><bool>true</bool></property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QRadioButton" name="radioButton_showOutside">
     <property name="text"><string>Show outside</string></property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="labelPolygonText">
     <property name="text"><string>Polygons:</string></property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_polygonCount">
     <property name="text"><string>0</string></property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_apply">
     <property name="text"><string>Apply</string></property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_deactivate">
     <property name="text"><string>Deactivate</string></property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButton_reset">
     <property name="text"><string>Reset</string></property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
